katcp/server.py

```diff
@@ -2217,16 +2226,20 @@ def request_sensor_sampling(self, req, msg):
             given. If the event occurs more than once within the minimum period,
             only one update will occur. Whether or not the event occurs, the
             sensor value will be updated at least once per maximum period.
-            The differential-rate strategy is not supported in this release.
+            For the differential-rate strategy there are 3 parameters. The first is the
+            same as the differential strategy parameter. The second and third are the
+            minimum and maximum periods, respectively, as with the event-rate strategy.
 
         Returns
         -------
         success : {'ok', 'fail'}
             Whether the sensor-sampling request succeeded.
-        name : str
-            Name of the sensor queried or configured.
-        strategy : {'none', 'auto', 'event', 'differential', 'period'}
-            Name of the new or current sampling strategy for the sensor.
+        names : str
+            Name(s) of the sensor queried or configured. If multiple sensors, this will
+            be a comma-separated list.
+        strategy : {'none', 'auto', 'event', 'differential', 'differential-rate',
+                    'period', 'event-rate'}.
+            Name(s) of the new or current sampling strategy for the sensor(s).
```

Comment:
Typo.  There is only one strategy that will be returned.
```suggestion
            Name of the new or current sampling strategy for the sensor(s).
```